javalib/src/main/scala/java/net/AbstractPlainSocketImpl.scala

```diff
@@ -215,43 +214,17 @@ private[net] abstract class AbstractPlainSocketImpl extends SocketImpl {
     if (timeout > 0)
       tryPollOnAccept()
 
-    val storage = stackalloc[Byte](sizeof[in.sockaddr_in6])
-    val len = stackalloc[socket.socklen_t]()
-    !len = sizeof[in.sockaddr_in6].toUInt
+    val ss = stackalloc[socket.sockaddr_storage]()
```

Comment:
Anybody of a certain age and ethnicity or sensibility will
object to a variable called ss.  Why not spend a few 
characters and give it a proper name?